extensions/vscode/webviews/homeView/src/components/views/projectFiles/TreeProjectFiles.vue

```diff
@@ -24,7 +24,7 @@
       #postDecor
       v-if="
         file.isFile &&
-        file.reason?.source !== FileMatchSource.BUILT_IN &&
```

Comment:
If the file is included it isn't `FileMatchSource.BUILT_IN` so we can reduce the if here.